time-sync-example/gke/cm-entrypoint.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: entrypoint
+  labels:
+    app: default-init
+data:
+  entrypoint.sh: |
```

Comment:
Instead of hardcoding this script in a ConfigMap, I suggest moving this in an `entrypoint.sh` file, and creating the config map on the fly from that file. So you get linting and checks running on this script as well.

https://kubernetes.io/docs/reference/kubectl/generated/kubectl_create/kubectl_create_configmap/